samples/zephyr/run-tests.sh

```diff
@@ -191,6 +191,115 @@ pyocd commander -c reset
 echo "Expected result: hello1 runs"
 ok_yn
 
+echo '--------------------------------------------------------'
```

Comment:
Can you make comparable updates to samples/zephyr/mcutests/mcutests.go. The run-tests.sh is kind of deprecated, and at least all of my testing is done with the Go version.

Reply:
Looks like the same feature has been merged in #1255 on 13 Jan (!)